src/map/quest.hpp

```diff
@@ -33,6 +33,8 @@ struct s_quest_objective {
 	e_race race;
 	e_size size;
 	e_element element;
+	std::string location;
+	std::string map_name;
```

Comment:
Is the mapname different from the "normal" mapname or can the existing name be used?
If it is the same I would not store it here.

Reply:
The "normal" name of the map is used like in mapInfo.lub, I have not seen another name.